site/src/pages/components/focusgroup.explainer.mdx

```diff
@@ -8,9 +8,9 @@ Authors: [Travis Leithead](https://github.com/travisleithead),
 [David Zearing](https://github.com/dzearing),
 [Chris Holt](https://github.com/chrisdholt/)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
```

Comment:
Why did you convert these all to MDX?

Reply:
Rehype seems to be applying to MDX only. I give reply to @gregwhitworth about heading.

Reply:
Thanks!